CHANGELOG.md

```diff
@@ -1,3 +1,14 @@
+## 0.56.0 / 2022-04-20
+
+* [CHANGE] Added validation at the API level for duration and time-based fields of the Prometheus CRD. #4684
```

Comment:
Missed validation for size based retention and log settings :) 
https://github.com/prometheus-operator/prometheus-operator/pull/4661
https://github.com/prometheus-operator/prometheus-operator/pull/4638

Reply:
thanks! not sure how I missed them :)